src/main/java/in/ashwanthkumar/gocd/github/provider/gitlab/GitLabProvider.java

```diff
@@ -121,7 +120,8 @@ private PullRequestStatus getPullRequestStatus(GitConfig gitConfig, String prId,
         } catch (Exception e) {
             // ignore
             LOG.warn(e.getMessage(), e);
-        } return null;
+            throw new RuntimeException(String.format("Failed to fetch PR status. %s", e.getMessage()), e);
```

Comment:
It would be nice to have the log and error have the same message so it is easier to grep on the logs when needed.